commands/bridge_pull.go

```diff
@@ -88,9 +121,13 @@ func runBridgePull(cmd *cobra.Command, args []string) error {
 	// send done signal
 	close(done)
 
-	fmt.Printf("Successfully imported %d issues and %d identities with %s bridge\n", importedIssues, importedIdentities, b.Name)
+	fmt.Printf("imported %d issues and %d identities with %s bridge\n", importedIssues, importedIdentities, b.Name)
 
-	return nil
+	// do not store last import time if context was cancelled
+	if ctx.Err() != context.Canceled && ctx.Err() != context.DeadlineExceeded {
```

Comment:
I'd say don't store the last import time if *any* error happened.

Reply:
Agreed. This will need mores checks for the event types